.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+__ULTIMATE PROBLEM:__
+
+
+
+__HOW IT WAS FIXED:__
+
+
+
+__TESTING SUGGESTIONS:__
+
+
+
+__POTENTIAL AREAS OF REGRESSION:__
```

Comment:
#supernit...

I think using markdown headers and normal casing makes this much more readable:
```markdown
## Ultimate problem:



## How it was fixed:



## Testing suggestions:



## Potential areas of regression:



---

__FYA:__ @greglittlefield-wf @aaronlademann-wf @jacehensley-wf @clairesarsam-wf @joelleibow-wf
```

> ## Ultimate problem:
>
>
>
> ## How it was fixed:
>
>
>
> ## Testing suggestions:
>
>
>
> ## Potential areas of regression:
>
>
> --- 
>
> __FYA:__ `@greglittlefield-wf` `@aaronlademann-wf` `@jacehensley-wf` `@clairesarsam-wf` `@joelleibow-wf`